test/test.ts

```diff
@@ -0,0 +1,73 @@
+import * as http from 'http';
```

Comment:
Have you consider to split our test into multiple test files? Is it possible for the framework to pick up all the test files?

Reply:
update package.json so that yarn test command can now run all tests file under test directory